### ISSUE TITLE
invalid argument and no rule to make target when making from examples

### ISSUE THREAD

**Issue by qinggengaoshen** [cocotb/cocotb#3027]
I am new to cocotb and python. when firstly making the adders in examples, I find several problems appearing in terminal

process_begin: CreateProcess(NULL, pwd, ...) failed.

'tr' is not recognized as an internal or external command,

D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:104:D:/hdl/D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.deprecations: Invalidargument.  -f shouldn't appear here 

D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:113:D:/hdl/D:\anaconda\lib\sitepackages\cocotb\share\makefiles/simulators/Makefile.: Invalid argument

make:*** No rule to make target  “D:/hdl/D:\anaconda\lib\site-packages\cocotb\share\makefiles/simulators/Makefile.” Stop

my system is Win10( 64 bits) and I use pycharm(64bits), aconda（64bits） and Python 3.6(pytorch) as python Interpreter.  I have already added the make-3.8.1 to Path. 
the make-3.8.1 built for i386-pc-mingw32. the simulator is Questa（64 bits,2021.1) with cocotb(1.6,2)

The problems bother me for a long time.  Can I know how to fix these problems  when making. 
Thanks

**Comment by qinggengaoshen** [comment]
Well. the problems is associated with the version of make. 
If I use the GNU make-3.8.1 built for pc-mingw32, the version and problems is as below :

D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:104:D:/hdl/D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.deprecations: Invalidargument. -f shouldn't appear here

D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:113:D:/hdl/D:\anaconda\lib\sitepackages\cocotb\share\makefiles/simulators/Makefile.: Invalid argument

But if I use GNU make 4.3 built for window32,the problems will change:
Makefile:34: pipe: No error
D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:113:D:/anaconda/lib/sitepackages/cocotb/share/makefiles/simulators/Makefile.: No such file or directory

Makefile is from the offical examples (adder)

**Comment by qinggengaoshen** [comment]
Finally, I did a lot on the software.  there are  less problems existing:
I press mingw32-make in the terminal and get:

'tr' is not recognized as an internal or external command.
-f shouldn't appear here.
D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:113:D:/anaconda/lib/sitepackages/cocotb/share/makefiles/simulators/Makefile.: No such file or directory
mingw32-make: *** No rule to make target 'D:/anaconda/lib/site-packages/cocotb/share/makefiles/simulators/Makefile.'.  Stop.

**Comment by qinggengaoshen** [comment]
@marlonjames
 I have already pressed the order: conda install -c msys2 m2-base m2-make before and  changed for GNU make 4.3 built for window32. But I found something wrong with the Makefile.sim in this environment. 
I press mingw32-make in the terminal and get:

'tr' is not recognized as an internal or external command.
-f shouldn't appear here.
D:\anaconda\lib\sitepackages\cocotb\share\makefiles/Makefile.sim:113:D:/anaconda/lib/sitepackages/cocotb/share/makefiles/simulators/Makefile.: No such file or directory
mingw32-make: *** No rule to make target 'D:/anaconda/lib/site-packages/cocotb/share/makefiles/simulators/Makefile.'. Stop.

Should I change some codes in the Makefile.sim?  If I am expected to do that, How should  i change it?
